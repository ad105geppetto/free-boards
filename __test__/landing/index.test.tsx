import "@testing-library/jest-dom";
import { act, render } from "@testing-library/react";
import Landing from "../../src/components/units/landing";

describe("랜딩 페이지 Unit Test", () => {
  it("랜딩 페이지 스냅샷 테스트", async () => {
    await act(async () => {
      const result = render(<Landing />);

      expect(result.container).toMatchSnapshot();
    });
  });
});
