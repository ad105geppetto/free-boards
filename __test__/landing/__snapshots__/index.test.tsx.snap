// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`랜딩 페이지 Unit Test 랜딩 페이지 스냅샷 테스트 1`] = `
<div>
  <div
    class="css-1ol1jm3"
  >
    <div
      class="css-aq3xj0"
    >
      <div
        class="css-bmik2f"
      >
        <img
          alt="transaction"
          class="css-j9letj"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/trading-icon.webp?alt=media"
        />
      </div>
      <div
        class="css-15rv61c"
      >
        <p
          class="css-1xtk7qn"
        >
          모두가 사용한 게시판
        </p>
        <p
          class="css-1xtk7qn"
        >
          요즘 게시판은 여기지!
        </p>
      </div>
      <div
        class="css-nvh3z5"
      >
        <div
          class="css-g90fw7"
        >
          Join us!
        </div>
        <img
          alt="down-arrow"
          class="css-1uh7s7m"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/down-arrow-icon.webp?alt=media"
        />
      </div>
      <ul
        class="css-pgji8g"
      >
        <li
          class="list-item css-1lwl0fi"
        >
          자유게시판
        </li>
        <li
          class="list-item css-d3d53p"
        >
          글쓰기
        </li>
        <li
          class="list-item css-ayrrkf"
        >
          좋은 글귀 추천
        </li>
        <li
          class="list-item css-d3d53p"
        >
          출석체크?
        </li>
        <li
          class="list-item css-1lwl0fi"
        >
          글을 수정하려면?
        </li>
        <li
          class="list-item css-ayrrkf"
        >
          좋은 게시판이였다!
        </li>
        <li
          class="list-item css-1lwl0fi"
        >
          자유게시판
        </li>
        <li
          class="list-item css-d3d53p"
        >
          글쓰기
        </li>
        <li
          class="list-item css-ayrrkf"
        >
          좋은 글귀 추천
        </li>
        <li
          class="list-item css-d3d53p"
        >
          출석체크?
        </li>
        <li
          class="list-item css-1lwl0fi"
        >
          글을 수정하려면?
        </li>
        <li
          class="list-item css-ayrrkf"
        >
          좋은 게시판이였다!
        </li>
        <li
          class="list-item css-1lwl0fi"
        >
          자유게시판
        </li>
        <li
          class="list-item css-d3d53p"
        >
          글쓰기
        </li>
        <li
          class="list-item css-ayrrkf"
        >
          좋은 글귀 추천
        </li>
        <li
          class="list-item css-d3d53p"
        >
          출석체크?
        </li>
        <li
          class="list-item css-1lwl0fi"
        >
          글을 수정하려면?
        </li>
        <li
          class="list-item css-ayrrkf"
        >
          좋은 게시판이였다!
        </li>
      </ul>
      <div
        class="css-gpqih7"
      >
        <img
          alt="social"
          class="image-icon css-4i3i7k"
          id="image_6"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/social.webp?alt=media"
        />
        <img
          alt="reporter-m"
          class="image-icon css-4i3i7k"
          id="image_4"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/reporter_m.webp?alt=media"
        />
        <img
          alt="freindship"
          class="image-icon css-4i3i7k"
          id="image_2"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/friendship.webp?alt=media"
        />
        <img
          alt="reporter-w"
          class="image-icon css-4i3i7k"
          id="image_5"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/reporter_w.webp?alt=media"
        />
        <img
          alt="author"
          class="image-icon css-4i3i7k"
          id="image_1"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/author.webp?alt=media"
        />
        <img
          alt="librarian"
          class="image-icon css-4i3i7k"
          id="image_3"
          loading="lazy"
          src="https://firebasestorage.googleapis.com/v0/b/used-item-boards.appspot.com/o/librarian.webp?alt=media"
        />
      </div>
    </div>
  </div>
</div>
`;
